Tic-Tac-Toe.py

```diff
@@ -0,0 +1,171 @@
+#Tic Tac Toe
+
+#create a 3x3 list of blank spaces
+def new_board():
+    board = [[" ", " ", " "], [" ", " ", " "], [" ", " ", " "]]
+    return board
+
+#print the current board to the user
+def print_board(board):
+    print("", board[0][0], "|", board[0][1], "|", board[0][2])
+    print("---+---+---")
+    print("", board[1][0], "|", board[1][1], "|", board[1][2])
+    print("---+---+---")
+    print("", board[2][0], "|", board[2][1], "|", board[2][2])
+
+#take the user input and return an X or O depending on who's turn it is
+def next_turn(turn):
+    if (turn % 2) == 0:
+        print("X's turn!")
+    elif (turn % 2) == 1:
+        print("O's turn!")
+    move = input("Enter your move: ")
+    print()
+    if turn%2 == 0:
+        return ("X", move)
+    else:
+        return ("O", move)
```

Comment:
Note: Totally up to you, but technically you don't need the parens here, python will turn it into a tuple for you.